### DISCUSSION TITLE
[Feature] Support specifying an instance to create a profile

### DISCUSSION THREAD

**Discussion by heihaozi** [apache/skywalking#7873] [General]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

When creating a profile, not only can all instances of the service be profiled, but an instance can also be specified.

### Use case

![](https://user-images.githubusercontent.com/2956927/136516739-d8673337-1ccf-48db-9acf-3b4033642605.png)

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
We don't expect that, from design perspective. Any SRE or Ops team is impossible to identify issue from one specific instance/pod.

**Comment by wu-sheng** [comment]
Move to discussion.

**Comment by heihaozi** [comment]
It has appeared in our production environment: only one instance has a relatively long response time, and the other instances are normal. 

Therefore, there are scenarios for profiling a specified instance, which may not affect other normal instances.

**Reply by wu-sheng** [reply]
This is more likely a debug tool, SRE team usually doesn't care onr pod.
